### PR TITLE
New webchat widget

### PR DESCRIPTION
**Pull request by martyn-w**
### Trello card
[Add new webchat widget](https://trello.com/c/T45plfcf/6147-add-new-webchat-widget)

### Context
Implementing a new webchat widget following a reprocurement exercise

### Changes proposed in this pull request
Webchat widget availability is now triggered from a 3rd-party API, rather than based solely on time/day

### Guidance to review
Webchat widget content is being developed by a 3rd party

### CODE REVIEW

File: app/controllers/chats_controller.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+private
+
+  def show_html
+    render layout: "chat"
+
+    # render \
+    #   template: "errors/not_found",
+    #   layout: "application",
+    #   status: :not_found
```
Comment (spencerldixon) [inline app/controllers/chats_controller.rb:26]: Stray comment

**Review by spencerldixon — DISMISSED** [review]
LGTM 👍

File: app/controllers/chats_controller.rb (hunk 1 above)
Comment (martyn-w) [reply app/controllers/chats_controller.rb:26]: Ooops! good spot 👍

**Review by spencerldixon — APPROVED** [review]
LGTM 👍

**Comment by martyn-w** [comment]
Merging now for testing on staging. It will remain deactivated on production for now.

**Pull request merged**